src/game_inventory.cpp

```diff
@@ -728,9 +728,12 @@ class comestible_inventory_preset : public inventory_selector_preset
                 if( you.has_trait( trait_SAPROPHAGE ) || you.has_trait( trait_SAPROVORE ) ) {
                     return 1;
                 } else {
-                    return 4;
+                    return 5;
                 }
             } else if( time == 0_turns ) {
+                return 4;
+            } else if( loc.has_parent() && loc.parent_item()->all_pockets_sealed() &&
+                       loc.parent_item()->get_all_contained_pockets().value().front()->spoil_multiplier() == 0.0f ) {
```

Comment:
```suggestion
            } else if( loc.has_parent() &&
                       loc.parent_item()->contained_where( *loc )->spoil_multiplier() == 0.0f ) {
```

Your check could return an incorrect value for items with multiple sealed pockets. I don't know if there are items like that in the game, but better safe than sorry.

Reply:
It occurs to me that there's also an `item::all_pockets_sealed()` function that could help simplify this.

Reply:
Since it's about the spoil multiplier of the pocket the item is in, `item::all_pockets_sealed()` isn't all that helpful.

Reply:
Oh duh, it was already there. Yeah you're right, it doesn't help to be sealed if spoil multiplier is nonzero.

Reply:
I am not 100% sure what the difference is between `loc.parent_item()->get_all_contained_pockets().value().front()->spoil_multiplier() == 0.0f` and `loc.parent_item()->contained_where( *loc )->spoil_multiplier() == 0.0f`. How did you simplify this? (Sorry this is my first PR and I am not 100% sure of the API)

Reply:
`get_all_contained_pockets` is a list of all the pockets of an item. `front` returns the first element in that list, which is fine if there is only one element but will be wrong if there are multiple pockets and the item is not in the first one. `contained_where` searches this list for the correct pocket containing this item instead.